irods/manager/data_object_manager.py

```diff
@@ -118,25 +118,25 @@ def get(self, path, local_path = None, num_threads = DEFAULT_NUMBER_OF_THREADS,
         Only download the object if the local_path is a string (specifying
         a path in the local filesystem to use as a destination file).
         """
-        parent = self.sess.collections.get(irods_dirname(path))
+        parent_name = irods_dirname(path)
 
         # TODO: optimize
         if local_path:
             self._download(path, local_path, num_threads = num_threads, **options)
 
-        query = self.sess.query(DataObject)\
+        query = self.sess.query(Collection,DataObject)\
             .filter(DataObject.name == irods_basename(path))\
-            .filter(DataObject.collection_id == parent.id)\
+            .filter(Collection.name == parent_name)\
             .add_keyword(kw.ZONE_KW, path.split('/')[1])
 
-        if self.sess.ticket__:
-            query = query.filter(Collection.id != 0) # a no-op, but necessary because CAT_SQL_ERR results if the ticket
+#       if self.sess.ticket__:
+#           query = query.filter(Collection.id != 0) # a no-op, but necessary because CAT_SQL_ERR results if the ticket
                                                      # is for a DataObject and we don't explicitly join to Collection
```

Comment:
This isn't needed anymore?

Reply:
Need to comment it back in apparently.  I was experimenting around  in that function and I guess I didn't undo this.... 
Yes, this code is needed still.